cumulus_library/study_parser.py

```diff
@@ -336,7 +332,7 @@ def _load_and_execute_builder(
 
         As with eating an ortolan, you may wish to cover your head with a cloth.
         Per an article on the subject: "Tradition dictates that this is to shield
-        – from God’s eyes – the shame of such a decadent and disgraceful act."
+        - from God's eyes - the shame of such a decadent and disgraceful act."
```

Comment:
**wat.** If `ruff` made you take out the PROPER unicode apostrophe and replace it with this tragically vertical approximation of an apostrophe, it's not fit for purpose!

\<small\>it's fine...\</small\>